packages/volto/locales/ca/LC_MESSAGES/volto.po

```diff
@@ -4873,6 +4965,11 @@ msgstr "Carregant"
 msgid "log in"
 msgstr "Iniciar Sessió"
 
+#. Default: "login"
+#: helpers/MessageLabels/MessageLabels
+msgid "login"
+msgstr ""
+
```

Comment:
Just to clarify, "Log in" is the action of logging in (and is the previous translation message), whereas "Login" is usually the username. I'm not sure why this one was added, and I need more context to be clear.

```suggestion
#. Default: "Login"
#: helpers/MessageLabels/MessageLabels
msgid "Login"
msgstr ""
```